app/blobs/templates.json

```diff
@@ -0,0 +1,23 @@
+[
+    { 
+        "imageURL": "images/barchart.png",
+        "templateName" : "Bar Chart",
+        "data" : {
+            "columns": [
+                ["data1", 30, 200, 100, 400, 150, 250],
+                ["data2", 50, 20, 10, 40, 15, 25]
+            ]
+        },
+        "dims" : 2,
+        "id" : 1
+
+    },
+    { 
+        "imageURL" : "images/linegraph.png",
+        "templateName" : "Line Graph",
+        "data" : {"test": 456},
+        "dims" : 2,
+        "id" : 2
```

Comment:
Is ID how we want to keep track of this? It seems fine to me - but would like the input of others.